app/queries/screening_list/query.rb

```diff
@@ -30,25 +31,29 @@ def generate_query(json)
           end if @q
 
           if @name
-            generate_fuzzy_queries(json, %w(name.keyword alt_names.keyword), @name)
+            # generate_fuzzy_queries(json, %w(name.keyword alt_names.keyword), @name)
+            generate_fuzzy_queries(json, %w(name alt_names), @name)
+          end
+          if @phonetics == '1'
+            generate_fuzzy_queries(json, ['phonetic_names'], @name)
```

Comment:
We don't have coverage for this line.

![screen shot 2015-04-03 at 3 45 12 pm](https://cloud.githubusercontent.com/assets/277504/6979584/74e7175e-da18-11e4-8b0b-1f5de0f616e2.png)